### ISSUE TITLE
Docs: Add docs and useful info for debugging CODAL, including with MakeCode & MicroPython

### ISSUE THREAD

**Issue by microbit-carlos** [lancaster-university/codal-microbit-v2#187]
Flags to enable for debug prints, how to enable those flags in MakeCode and MicroPython builds, etc.

We were considering having this information in the MakeCode and MicroPython documentation as well, but ultimately this is usually necessary when debugging from the CODAL layer, so this would be a good place to have them. And it'd be more efficient to have this in a repo like this one, rather than trying to find the "right place" (which might not exists yet) in MakeCode/MicroPython docs.

**Comment by JohnVidler** [comment]
Agreed, I'll put this on my documentation ToDo for additional information to include.
